### ISSUE TITLE
New Source: Dremio

### ISSUE THREAD

**Issue by RealChrisSean** [airbytehq/connector-contest#116]
https://github.com/airbytehq/airbyte/issues/9031

**Comment by adi-uchiha** [comment]
Would love to work on this... Can u guide me further

**Comment by RealChrisSean** [comment]
Hello @adi-uchiha we have a getting started guide [here](https://docs.airbyte.com/connector-development/config-based/tutorial/getting-started). 

We also have a [video tutorial ](https://www.youtube.com/watch?v=i7VSL2bDvmw) that you can watch. Part 2 will go live on Monday :).

If you join our slack channel, two of our engineers are offering 30-minute sessions every week (during hacktober) to help guide anyone with their connector. You can join the #hacktobestfest-2022 slack channel [here](https://app.slack.com/accept-shared-channel/T02TET08MDH/I043WFREPQX/enQtNDEzMjUzNTQ5OTg0Ny0xN2I2MDU4NGFkOTNlYzZlN2JmMGIzOTgyMjU1ODk1MDIzZWM5YTAzMThkMDFkNzgzNWNjNzljOGU1MmRhMDIw).

I hope this helps! Did you want me to assign this issue to you?

**Comment by adi-uchiha** [comment]
@RealChrisSean sure. I look forward towards learning and working on this. Thanks for the reference.

**Comment by RealChrisSean** [comment]
Excellent good luck @adi-uchiha :).

**Comment by RealChrisSean** [comment]
Hi @adi-uchiha how has this been going for you? Is there anything we can assist you with?

**Comment by adi-uchiha** [comment]
Hey @RealChrisSean thanks for reaching out 
I was having trouble building connector. 
Tried a lot to solve that on my own.
I posted that on slack 7 days ago. Got no reply. 
https://airbytehq.slack.com/archives/C027KKE4BCZ/p1665322636228029
I felt that community support is weak. I cannot proceed without solving that problem.
Would be great if you assign me mentor or something like that whom I can reach out when I have trouble as I new to this.
